src/clr-addons/notification/notification.scss

```diff
@@ -0,0 +1,63 @@
+@include exports('notification.clr-addons') {
+  .notification {
+    &__wrapper {
+      position: fixed;
+      z-index: 1000;
+      top: 36px;
+      width: 400px;
+      left: 90px;
+      right: 0;
+      border-radius: 4px;
+      margin: 0 auto;
+      overflow: hidden;
+    }
+    .progress,
+    .progress-static {
+      height: 0.5em;
+      position: absolute;
+      z-index: 1;
+    }
+    .alert-items {
+      padding: 0.333333rem calc(0.75rem - 1px);
+    }
+    .alert-app-level {
+      .alert-items {
+        padding-top: 0.76rem;
+        padding-bottom: 0.86rem;
+
+        .alert-item {
+          justify-content: left;
+        }
+      }
+      .alert-icon-wrapper {
+        margin-top: 0.17rem;
+        height: 1.2rem;
+      }
+      .close {
+        height: 3rem;
+        width: 2rem;
+
+        clr-icon {
+          margin-top: -0.268333rem;
+        }
+      }
+    }
+    .alert {
```

Comment:
this block can be merged with the .alert-app-level style above, as .alert and .alert-app-level are always present